### DISCUSSION TITLE
Displayable image files to .heic format

### DISCUSSION THREAD

**Discussion by alexey1312** [community/community#10258] [Pull Requests]
Hey, Apple added support for High Efficiency Image File Format (HEIC) since the following OS versions:
iOS 11.0+
macOS 10.13+
Mac Catalyst 13.0+
tvOS 11.0+
watchOS 4.0+
Is there any plan to support this format in the near future?

**Comment by yanz-safe** [comment]
Bumping this thread.
We are using snapshot tests on iOS, and to minimize the size of images, we decided to use the `.heic` format. Unfortunately, these images are not displayed in GitHub Web UI, so it makes hard to review the snapshot changes from GitHub Pull Request UI :(
